src/lumigo_tracer/utils.py

```diff
@@ -67,6 +75,16 @@ def config(
     Configuration.is_step_function = step_function
     Configuration.timeout_timer = timeout_timer
     Configuration.send_only_if_error = os.environ.get("SEND_ONLY_IF_ERROR", "").lower() == "true"
+    if "LUMIGO_DOMAINS_SCRUBBER" in os.environ:
+        try:
+            Configuration.domains_scrubber = json.loads(os.environ["LUMIGO_DOMAIN_SCRUBBER"])
```

Comment:
Are you sure this is in json format?

Reply:
yep,
and that's what I demand in the readme...

do you have another idea?